src/main/scala/com/cloudera/sparkts/models/ARIMA.scala

```diff
@@ -211,7 +211,7 @@ object ARIMA {
    * @param includeIntercept flag to include intercept
    * @return initial ARMA(p, d, q) parameter estimates
    */
-  private def hannanRissanenInit(
+  def hannanRissanenInit(
```

Comment:
Can you make this `private[models]` so it's available to ARIMAX, but not exposed as a public API?